### ISSUE TITLE
Swate "Pick directories" not funktional

### ISSUE THREAD

**Issue by frsommer** [nfdi4plants/Swate#518]
### Contact Details

frsommer

### What happened?

Tried to pick directories in Swate/ARCitect
Button does not work, no reaction.
The file picker works...
![image](https://github.com/user-attachments/assets/f27c62d0-e4f8-4e43-afe1-6c374884ef21)

### What did you expect?

select directory and insert path

### Host

Applies to All

### Host Details

Arcitect 0,41
swate 1000 8beta
Win10

### Swate version

1.0.0.beta.08

**Comment by frsommer** [comment]
just two more comments:
1. does not work when using the file picker from the ArCitect menu as well.
2. maybe rephrase "...insert their names into Excel" Excel to SWATE?
![image](https://github.com/user-attachments/assets/5fc114de-72c0-48f1-8196-6d9d690fe06b)

**Comment by ZimmerD** [comment]
Can replicate this using Swate Release Version [v1.0.0-beta.11](https://github.com/nfdi4plants/Swate/releases) while running the ARCitect

**Comment by Freymaurer** [comment]
@ZimmerD @frsommer 

Hey sorry for the late investigation. This was actually an ARCitect issue, will fix it there 👍

**Comment by Freymaurer** [comment]
Fixed and done!
